### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by itscaro**
## Description

- Add instructions to install on Windows
- Add `go install` way
- Add Go 1.9.x
- Separate homebrew section to guide/homebrew.md
- Add scoop section with link to guide/scoop.md & how to update scoop manifest
- Fix ordered list

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation changes

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ This CLI can build and deploy templated functions, so it's best if you have FaaS
 
 > Here's how to install Go in 60 seconds.
 
-* Grab Go 1.7.x or 1.8.3 from https://golang.org/dl/
+* Grab Go 1.7.x, 1.8.3 or 1.9.x from https://golang.org/dl/
```
Comment (alexellis) [inline CONTRIBUTING.md]: Let's remove 1.7.x from here?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -69,33 +69,10 @@ Use the tool if you add new dependencies or want to update the existing ones.
...
-
-After `brew edit` run the build and test the results:
-
-```
-$ brew uninstall --force faas-cli ; \
-  brew install --build-from-source faas-cli ; \
-  brew test faas-cli ; \
-  brew audit --strict faas-cli
-```
+### How to update the `scoop` manifest
```
Comment (alexellis) [inline CONTRIBUTING.md]: How popular is `scoop`? Most of us hadn't heard of it.

File: README.md [hunk 3]
```diff
@@ -31,7 +33,30 @@ $ brew install faas-cli
...
+Note: The `scoop` release may not run the latest minor release but is updated regularly.
+
+or directly with PowerShell:
+
+```
+$version = (Invoke-WebRequest "https://api.github.com/repos/openfaas/faas-cli/releases/latest" | ConvertFrom-Json)[0].tag_name
+(New-Object System.Net.WebClient).DownloadFile("https://github.com/openfaas/faas-cli/releases/download/$version/faas-cli.exe", "faas-cli.exe")
+```
+
+#### Get with GO
```
Comment (alexellis) [inline README.md]: Don't think we need this paragraph.

File: README.md [hunk 4]
```diff
@@ -17,6 +17,8 @@ Demo: [ASCII cinema](https://asciinema.org/a/141284)
 
 ### Get started: Install the CLI
 
+#### Unix-based
```
Comment (alexellis) [inline README.md]: Keep the Windows heading, but no need for the default heading to be added.

**Comment by alexellis** [comment]
Along with other comments - just to mention `go install` (mentioned in description) is probably not going to do the right thing re: ldflags / injected variables.

How widely used is `scoop`?

**Comment by alexellis** [comment]
@itscaro hope you had a good New Year and thanks for all the help in 2017. Please could you address comments?

File: CONTRIBUTING.md (hunk 2 above)
Comment (itscaro) [reply CONTRIBUTING.md]: I don't know, between chocolatey and scoop, the stats of stars, fork, watching are likely the same. I used chocolatey before but now I prefer scoop

**Comment by itscaro** [comment]
Sorry, I missed the update mails on this PR

**Pull request merged**